src/components/exchange/ExchangeModalHeader.js

```diff
@@ -39,7 +39,6 @@ const InfoButton = styled(Centered).attrs({
   pressOutTension: 300,
   useNativeDriver: true,
 })`
-  ${InfoButtonPosition};
```

Comment:
Are you sure you wanna remove this for iOS too?

Reply:
Pretty sure yes because `InfoButtonPosition` was used multiple times.
I removed it from `InfoButton` but it still is present in styles for `InfoButtonTransition` and below we can see that InfoButtonTransition is just wrapper for InfoButton so it should be good 

And I tested it on iOS and looked good ;-;